test/recipes/controls/aws_parallelcluster_install/directories_spec.rb

```diff
@@ -0,0 +1,39 @@
+control 'setup_dirs' do
+  title 'Setup of basic directories'
+
+  base_dir = "/opt/parallelcluster"
+
+  describe directory(base_dir) do
```

Comment:
is it possible to do a cycle given that all the tests are the same?

Reply:
good point, I'll take a look